core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -302,7 +311,7 @@ public void notifyNotLeader(Collection<TransactionContext> pendingEntries) {
   @Override
   public void notifyTermIndexUpdated(long term, long index) {
     super.notifyTermIndexUpdated(term, index);
-    CompletableFuture.runAsync(mJournalSystem::updateGroup);
+    CompletableFuture.runAsync(mJournalSystem::updateGroup, mJournalPool);
```

Comment:
Just curious is the state of the excution not important? do we care if the updateGroup call failed?

Reply:
`mJournalSystem` falls back to the group information it held previously in case `updateGroup` fails. I think this is the best we can do if it fails, as we don't have a way of inferring group information.